package.json

```diff
@@ -44,7 +44,8 @@
     ]
   },
   "dependencies": {
-    "noble": "^1.9.0"
+    "noble": "^1.9.0",
+    "noble-mac": "git+https://github.com/Timeular/noble-mac.git"
```

Comment:
unfortunately version 0.0.3 is not yet published to npm, so in the meantime we have to install `noble-mac` from the git repo